yarn.lock

```diff
@@ -308,7 +308,7 @@
 
 "@grpc/grpc-js@^1.8.14":
   version "1.8.21"
-  resolved "https://registry.yarnpkg.com/@grpc/grpc-js/-/grpc-js-1.8.21.tgz#d282b122c71227859bf6c5866f4c40f4a2696513"
```

Comment:
I don't think we should switch to yarn registry

Reply:
You mean switch from yarn registry? This PR is switching it to the NPM registry from the yarn registry.

Reply:
Was accident. Will revert `yarn.lock` changes.